opslevel/resource_opslevel_check_git_branch_protection.go

```diff
@@ -0,0 +1,71 @@
+package opslevel
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/opslevel/opslevel-go"
+)
+
+func resourceCheckGitBranchProtection() *schema.Resource {
+	return &schema.Resource{
+		Description: "Manages a tool usage check",
+		Create:      wrap(resourceCheckGitBranchProtectionCreate),
+		Read:        wrap(resourceCheckGitBranchProtectionRead),
+		Update:      wrap(resourceCheckGitBranchProtectionUpdate),
+		Delete:      wrap(resourceCheckDelete),
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: getCheckSchema(map[string]*schema.Schema{
+			"tool_category": {
```

Comment:
I think I'm missing where this is being used.  From looking at `getCheckSchema` it looks like it's being added to the check schema but required is true so does that mean it needs to be used somewhere?

Reply:
Great catch - i forgot to remove this stuff